src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/AbstractLogsTab.php

```diff
@@ -146,4 +165,13 @@ protected function getRequestLimit(): int
 
         return 20;
     }
+
+    protected function getRequestSort(): string
+    {
+        if (isset($_REQUEST['sort'])) {
+            return in_array($_REQUEST['sort'], ['asc', 'desc']) ? $_REQUEST['sort'] : 'desc';
+        }
```

Comment:
```suggestion
        if (isset($_REQUEST['sort']) && strtolower($_REQUEST['sort']) === 'asc' ) {
            return 'asc';
        }
```

Reply:
The `in_array...` is needed to check if the sort param is 'asc or desc' because it will thrown an SQL query if not